src/index.css

```diff
@@ -1,5 +1,51 @@
 body {
-  margin: 0;
-  padding: 0;
-  font-family: sans-serif;
-}
+    font: 14px "Century Gothic", Futura, sans-serif;
+    margin: 20px;
+  }
+  
+  ol, ul {
+    padding-left: 30px;
+  }
+  
+  .board-row:after {
+    clear: both;
+    content: "";
+    display: table;
+  }
+  
+  .status {
+    margin-bottom: 10px;
+  }
+  
+  .square {
```

Comment:
You want to co-locate this styles next to each component eventually, consider a refactor on which every component has its own folder with an `index.js` (that way the imports remain the same) which imports a local `./style.css` with the specific styles for the component

Reply:
Sounds like a nice file organization for css. Talking about tree structure I've heard some separate also presentational from container components in different folders. I've never did that separation in Ember but sounds like it would help organizing things in redux-land.

Reply:
Yes, definitively. I tend to have at least a `Components` folder (presentational) and a `Screens` folder (container + also usually have a route, in redux terms, the ones that have a `connect` decorator). 
Then, only if needed, sometimes I have a `Containers` folder for `connect`ed things that  are not screens (e.g. a user menu). 
Finally, and this is less usual, sometimes I also prefer to separate folder for `Layouts` (presentational but only deal with the disposition of other presentational components)